_projects/decrease_docker_image_size.md

```diff
@@ -0,0 +1,76 @@
+---
+collaborating_projects:
+  - docker-coala-base
+desc: "Find creative ways to decrease the coala Docker image size"
+issues:
+  - https://github.com/coala/docker-coala-base/issues/55
+difficulty: "hard"
+initiatives:
+  - GCI
+markdown: decrease_docker_image_size.md
+mentors:
+  - yukiisbored
+name: "Decrease Docker image size"
+tags:
+  - coala
+  - docker
+  - linux
+---
+
+## Task
+
+Submit a Pull Request to the [docker-coala-base][repo] project which decreases
+the image size.
+
+## Objective
+
+This task is considerably hard which requires creativity and knowledge for
```

Comment:
for -> of

Reply:
also maybe `this task is hard because it requires considerable <insert rest of sentence here>...`?